eng/pipelines/runtime-staging.yml

```diff
@@ -176,6 +176,33 @@ jobs:
         creator: dotnet-bot
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
 
+#
+# Build Mono and Libraries for Android using Android native crypto instead of OpenSSL
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - Android_x64
```

Comment:
Can we add a 32-bit platform here as well (like x86) for additional build validation?

Reply:
Oh hey, it hit the x86 build issue that you had to fix. Woot.